MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```
@@ -280,6 +280,19 @@ final class FeedUIIntegrationTests: XCTestCase {
 		}
 		wait(for: [exp], timeout: 1.0)
 	}
+    
+    func test_loadFeedCompletion_rendersErrorMessageOnErrorUntilNextReload() {
```

It's missing a test: should hide the error when the user taps on the error view.